aviatrix/resource_aviatrix_gateway.go

```diff
@@ -513,13 +502,24 @@ func resourceAviatrixGatewayCreate(d *schema.ResourceData, meta interface{}) err
 		log.Printf("[INFO] failed to create Aviatrix gateway: %#v", gateway)
 		return fmt.Errorf("failed to create Aviatrix gateway: %s", err)
 	}
-
 	d.SetId(gateway.GwName)
 
 	flag := false
 	defer resourceAviatrixGatewayReadIfRequired(d, meta, &flag)
 
-	// single_AZ enabled for Gateway. https://docs.aviatrix.com/HowTos/gateway.html#high-availability
+	enableVpnNat := d.Get("enable_vpn_nat").(bool)
+	if vpnStatus {
+		if !enableVpnNat {
+			err := client.DisableVpnNat(gateway)
+			if err != nil {
+				return fmt.Errorf("failed to disable VPN NAT: %s", err)
+			}
+		}
+	} else if !enableVpnNat {
+		return fmt.Errorf("'enable_vpc_nat' is only supported for vpn gateway. Can't disable it")
```

Comment:
disable VPN NAT

Reply:
'enable_vpc_nat' is only for vpn gateway. Can not be set as false for non-van gateway.